asf_core_data/getters/data_getters.py

```diff
@@ -229,11 +228,27 @@ def load_s3_data(
     """
 
     if fnmatch(file_name, "*.xlsx"):
-        data = pd.read_excel(
-            os.path.join("s3://" + bucket_name, file_name),
-            sheet_name=None,
-            dtype=dtype,
-        )
+        excel_file = pd.ExcelFile(os.path.join("s3://" + bucket_name, file_name))
+        sheet_names = excel_file.sheet_names
+
+        # if excel file has data in multiple sheets
+        if len(sheet_names) > 1:
+            data = pd.DataFrame()
+            for sheet in sheet_names:
+                aux = pd.read_excel(
+                    os.path.join("s3://" + bucket_name, file_name),
+                    sheet_name=sheet,
+                    dtype=dtype,
+                )
```

Comment:
Looks good, but you can probably get around the aux with something like this (please test/double-check). That should be less computationally expensive than using aux and several concats.

```suggestion
            data = pd.concat([pd.read_excel(
                    os.path.join("s3://" + bucket_name, file_name),
                    sheet_name=sheet,
                    dtype=dtype,
                ) for sheet in sheets])
```

Reply:
Have implemented Solomon's suggestion instead, which makes the code a bit shorter and also we only need to load the file once :-)